src/main/resources/anniv6Resources/shaders/BrokenSpace/Glitch.fs

```diff
@@ -0,0 +1,115 @@
+
```

Comment:
I checked this PR out locally because I wanted to see what some of the relics do, and the shaders that are applied to the relics need to be addressed. I'm not particularly sensitive to color/motion and I found having a relic using the shaders (specifically Broken Snecko Eye) on screen for a combat to be really obnoxious, almost nauseating. I would hate having that in my relic bar for a full run; I'd probably abandon the run or console remove the relic. 

Could you tone down this effect? I would suggest entirely or almost entirely eliminating the movement and color changes. You can have a distorted image of the relic, that _very slowly_ shifts color and changes how its distorted. If it constantly draws the eye to it because it's obviously moving and changing a way that nothing else on screen is, that's too distracting.

Reply:
The relic shader is better now, but still very distracting.

Could you please make a no movement version? I understand that your intent with this is to making something interesting and cool, but seeing this in action, I've started to think that it's best to avoid having things in your relic bar move at all.

To clarify where we might be interpreting things differently, when I suggested having it move "very slowly" in my previous comment, I meant "so slow that it is almost imperceptible". Flipping around every few seconds is still too much. Thus the ask for a no movement version; that's a clear baseline we can start from. You can have the shader distort the base image by an amount that doesn't change, or only changes on save/reload or something like that.

Reply:
Thanks, I'm much more of a fan of the new, non-moving version. The distortion effect is cool but the relic is still vaguely recognizable, and it's no longer so distracting.